nemo/collections/nlp/modules/common/megatron/fused_softmax.py

```diff
@@ -66,9 +66,72 @@ def forward_torch_softmax(self, input, mask):
 
 else:
 
-    class MatchedScaleMaskSoftmax(ApexGuardDefaults):
-        def __init__(self):
+    class MatchedScaleMaskSoftmax(torch.nn.Module):
```

Comment:
Is this copy pasted from Apex? there should be a unittest that checks for equivalence between nemo and apex then

Reply:
Yes it's copied from apex, do you or @ericharper have a example unittest somewhere I can follow?